Specta/Specta/SPTIncludeGlobalBeforeAfterEach.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2015 Specta Team. All rights reserved.
+ */
+#import <Foundation/Foundation.h>
+
+// This protocol is used for whitelisting classes for global beforeEach and afterEach blocks.
+// If you want a class to participate in those just add this protocol to a category and it will be
+// included.
+@protocol SPTIncludeGlobalBeforeAfterEach <NSObject>
```

Comment:
A nitpick, but I think this reads better as `SPTGlobalBeforeAfterEach`. `Exclude` made sense, `Include` seems redundant.